litmus-portal/frontend/src/pages/WorkflowAnalytics/index.tsx

```diff
@@ -0,0 +1,386 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Here also, you can remove the eslint disable commands

Reply:
This page will be refactored it will be updated after this beta.